imports/ui/AddAccountingEntry.jsx

```diff
@@ -30,17 +32,17 @@ export const AddAccountingEntry = ({ format, id }) => {
   const formRef = useRef();
   const [accountId, setAccountId] = useState(null);
 
-  const { categories, accounts, accountsMap, existingEntry } = useTracker(
-    () => {
-      const accountsData = Accounts.find().fetch();
-      return {
-        categories: Categories.find().fetch(),
-        accounts: accountsData,
-        accountsMap: keyBy(accountsData, '_id'),
-        existingEntry: AccountingEntries.findOne(id),
-      };
+  const { categories, accounts, accountsMap, existingEntry, subscriptionHandle } = useTracker(() => {
+    const handle = Meteor.subscribe("newAccounting.fetchAll");
+    const accountsData = Accounts.find().fetch();
+    return {
+      subscriptionHandle: handle,
```

Comment:
You should never return the subscriptionHandle. If the subscription is not ready yet, you should have a flag like:

loading: true, loading: false, that you return from the tracker.